cmd/root.go

```diff
@@ -109,6 +111,7 @@ func initLogger() {
 func initConfig() {
 	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
 
+	viper.SetDefault("parallel", -1)
```

Comment:
The BindPFlag should take care of things.  Is this specifically for the `init` subcmd?